kubeflow-pipelines.yaml

```diff
@@ -62,29 +59,32 @@ pipeline:
     with:
       patches: kfp.patch
 
+  - uses: patch
+    with:
+      patches: upgrade-urllib3.patch
+
+  - uses: patch
+    with:
+      patches: upgrade-urllib3-for-backend.patch
+
+  - uses: patch
+    with:
+      patches: add-overrides.patch
+
 subpackages:
   - range: backends-go-builds
     name: "kubeflow-pipelines-${{range.key}}"
     description: "Kubeflow Pipelines ${{range.key}}"
     pipeline:
       - runs: |
           cd backend
-          # Remediate GHSA-p782-xgp4-8hr8
-          go get golang.org/x/sys@v0.0.0-20220811171246-fbc7d0a398ab
-
-          # Mitigate GHSA-69ch-w2m2-3vjp
-          go get golang.org/x/text@v0.9.0
-          # GHSA-vvpx-j8f3-3w6h
-          go get golang.org/x/net@v0.7.0
-
-          go get golang.org/x/mod@v0.12.0
-
-          # Mitigate GHSA-gwc9-m7rh-j2ww
-          # Mitigate GHSA-8c26-wmh5-6g9v
-          go get golang.org/x/crypto@1baeb1ce4c0b
-
           # GHSA-p782-xgp4-8hr8
           go get golang.org/x/sys@v0.8.0
+          go get golang.org/x/text@v0.9.0
```

Comment:
Looks like this used to be commented with #GHSA-69ch-w2m2-3vjp, but now it's not. Any reason? It doesn't look to be part of GHSA-p782-xgp4-8hr8.

Likewise for the others here?

Reply:
I removed it by an accident, re-added, thank you!